engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/WorkflowExecutionActor.scala

```diff
@@ -176,10 +176,10 @@ final case class WorkflowExecutionActor(workflowId: WorkflowId, workflowDescript
       goto(WorkflowExecutionFailedState) using stateData.mergeExecutionDiff(WorkflowExecutionDiff(Map(jobKey -> ExecutionStatus.Failed)))
     case Event(BackendJobExecutionSucceededResponse(jobKey, callOutputs), stateData) =>
       handleCallSuccessful(jobKey, callOutputs, stateData)
-    case Event(BackendJobExecutionFailedResponse(jobKey, reason), stateData) =>
+    case Event(BackendJobExecutionFailedNonRetryableResponse(jobKey, reason, _), stateData) =>
```

Comment:
ToL: I think I prefer `BackendJobExecutionActor.FailedNonRetryableResponse` to the monolithic and batholithic `BackendJobExecutionFailedNonRetryableResponse `